response_selectors/ranking_based_response_selector/component.yml

```diff
@@ -0,0 +1,100 @@
+ranking-based-response-selector-service:
+  name: response_selector
+  display_name: Response Selector
+  container_name: ranking-based-response-selector-service
+  component_type: null
+  model_type: Dictionary/Pattern-based
+  is_customizable: false
+  author: DeepPavlov
+  description: Algorithm that selects a final responses among the given list of candidate
+    responses via the given ranking service
```

Comment:
Algorithm that selects the final response among the given list of candidate
    responses via the given ranking service